src/pages/Money/Feed.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { connect, MapStateToProps } from "react-redux";
+
+import { AppState } from "../../store";
+import { Member } from "../../reducers/membersNew";
+import { Operation, OperationType } from "../../reducers/operations";
+import { getMemberOperations } from "../../selectors/operations";
+
+import OperationList from "../../components/OperationList";
+
+interface OwnProps {
+  loggedInMember: Member;
+}
+
+interface StateProps {
+  operations: Operation[];
+}
+
+type Props = OwnProps & StateProps;
+
+const FeedView: React.StatelessComponent<Props> = props => {
+  const { operations } = props;
+  return (
+    <section>
+      <h1>Your activity</h1>
```

Comment:
👍

Reply:
I was sort of thinking that it would always be your own feed... but I guess that could be a little confusing. Updated to be the feed of whoever you're looking at.